### PR TITLE
Fix bad error messaging when JS package ownership fails

### LINKED ISSUE #15: Running with minimal config errors
Gemfile
```
gem "code_ownership", "~> 1.28", require: false
```

config/code_ownership.yml
```
owned_globs:
  - app/models/**/*.rb
```

Running `bundle exec codeownership validate` produces this error:

```
bundler: failed to load command: codeownership (/home/ruby/apps/buyer_app/vendor/bundle/ruby/2.7.0/bin/codeownership)
Traceback (most recent call last):
	55: from /home/.asdf/installs/ruby/2.7.5/bin/bundle:23:in `<main>'
	54: from /home/.asdf/installs/ruby/2.7.5/bin/bundle:23:in `load'
	53: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/exe/bundle:36:in `<top (required)>'
	52: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/lib/bundler/friendly_errors.rb:103:in `with_friendly_errors'
	51: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/exe/bundle:48:in `block in <top (required)>'
	50: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/lib/bundler/cli.rb:25:in `start'
	49: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	48: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/lib/bundler/cli.rb:31:in `dispatch'
	47: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	46: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	45: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	44: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/lib/bundler/cli.rb:484:in `exec'
	43: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/lib/bundler/cli/exec.rb:23:in `run'
	42: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/lib/bundler/cli/exec.rb:58:in `kernel_load'
	41: from /home/.asdf/installs/ruby/2.7.5/lib/ruby/gems/2.7.0/gems/bundler-2.3.4/lib/bundler/cli/exec.rb:58:in `load'
	40: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/bin/codeownership:23:in `<top (required)>'
	39: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/bin/codeownership:23:in `load'
	38: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/bin/codeownership:5:in `<top (required)>'
	37: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/cli.rb:11:in `run!'
	36: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/cli.rb:63:in `validate!'
	35: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/_methods.rb:270:in `block in _on_method_added'
	34: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `validate_call'
	33: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `bind_call'
	32: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership.rb:59:in `validate!'
	31: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/_methods.rb:270:in `block in _on_method_added'
	30: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `validate_call'
	29: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `bind_call'
	28: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private.rb:44:in `validate!'
	27: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private.rb:44:in `flat_map'
	26: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private.rb:44:in `each'
	25: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private.rb:45:in `block in validate!'
	24: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/_methods.rb:270:in `block in _on_method_added'
	23: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `validate_call'
	22: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `bind_call'
	21: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private/validations/files_have_owners.rb:14:in `validation_errors'
	20: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/_methods.rb:270:in `block in _on_method_added'
	19: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `validate_call'
	18: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `bind_call'
	17: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private.rb:111:in `files_by_mapper'
	16: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private.rb:111:in `each'
	15: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private.rb:112:in `block in files_by_mapper'
	14: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/_methods.rb:270:in `block in _on_method_added'
	13: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `validate_call'
	12: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `bind_call'
	11: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private/ownership_mappers/js_package_ownership.rb:32:in `map_files_to_owners'
	10: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/_methods.rb:270:in `block in _on_method_added'
	 9: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `validate_call'
	 8: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation.rb:161:in `bind_call'
	 7: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private/parse_js_packages.rb:53:in `all'
	 6: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private/parse_js_packages.rb:53:in `map'
	 5: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private/parse_js_packages.rb:54:in `block in all'
	 4: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation_2_7.rb:106:in `block in create_validator_method_fast1'
	 3: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/sorbet-runtime-0.5.10172/lib/types/private/methods/call_validation_2_7.rb:106:in `bind_call'
	 2: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/code_ownership-1.28.0/lib/code_ownership/private/parse_js_packages.rb:23:in `from'
	 1: from /home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/json-2.3.1/lib/json/common.rb:263:in `parse'
/home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/json-2.3.1/lib/json/common.rb:263:in `parse': 784: unexpected token at '{ (JSON::ParserError)
'
```

### PR DESCRIPTION
**Pull request by AlexEvanczuk**
Fixes #15

### CODE REVIEW

File: lib/code_ownership/private/parse_js_packages.rb [hunk 1]
```diff
@@ -32,6 +32,14 @@ def self.from(pathname)
             name: package_name,
             metadata: package_loaded_json[METADATA] || {}
           )
+        rescue JSON::ParserError
```
Comment (technicalpickles) [inline lib/code_ownership/private/parse_js_packages.rb:35]: What is the message from this? Might be useful to include in the message.

Comment (AlexEvanczuk) [reply lib/code_ownership/private/parse_js_packages.rb:35]:
```
/home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/json-2.3.1/lib/json/common.rb:263:in `parse': 784: unexpected token at '{ (JSON::ParserError)
```

(from the issue above).

Comment (technicalpickles) [reply lib/code_ownership/private/parse_js_packages.rb:35]: Cool, so maybe add `=> e` to the rescue, and include #{e.message} on the first line?

Comment (AlexEvanczuk) [reply lib/code_ownership/private/parse_js_packages.rb]: Done!

**Review by technicalpickles — APPROVED** [review]

**Pull request merged**